exporter/elasticsearchexporter/integrationtest/datareceiver.go

```diff
@@ -217,6 +217,7 @@ func (es *mockESReceiver) Start(ctx context.Context, host component.Host) error
 	r.HandleFunc("/_bulk", func(w http.ResponseWriter, r *http.Request) {
 		if !es.config.DecodeBulkRequests {
 			w.WriteHeader(http.StatusOK)
+			w.Write([]byte("{}"))
```

Comment:
This line is causing a [lint failure](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/9504158221/job/26196208897?pr=33554):
```
Error: integrationtest/datareceiver.go:220:11: Error return value of `w.Write` is not checked (errcheck)
level=info msg="File cache stats: 7 entries of total size 32.1KiB"
level=info msg="Memory: 187 samples, avg is 1196.6MB, max is 1527.6MB"
level=info msg="Execution took 21.433686086s"
			w.Write([]byte("{}"))
			       ^
make[2]: *** [../../../Makefile.Common:199: lint] Error 1
make[1]: *** [Makefile:179: exporter/elasticsearchexporter/integrationtest] Error 2
```

Reply:
It is fixed now, thanks for the details :)